docs/docs/adrs/adr-018-permissionless-ics.md

```diff
@@ -0,0 +1,304 @@
+---
+sidebar_position: 19
+title: Permissionless ICS
+---
+# ADR 18: Permissionless Interchain Security
+
+## Changelog
+* 27th of June, 2024: Initial draft
+
+## Status
+
+Proposed
+
+## Context
+Currently, a consumer chain can join _Interchain Security_ (ICS) only through a [governance proposal](../features/proposals.md).
+A governance proposal was needed before the introduction of [Partial Set Security](../features/partial-set-security.md) (PSS)
+because validators were required to validate a consumer chain. However, after the introduction of PSS, a consumer chain can
+be either _Top N_ or _Opt In_. If a chain is an Opt In chain, then no validator is required to validate this chain unless they choose to.
+Because of this, we can launch an Opt In consumer chain without going through a governance proposal.
+
+This ADR presents _Permissionless_ ICS, a way in which an [_Opt In_](adr-015-partial-set-security.md) consumer chain can join
+ICS without needing a governance proposal but by simply issuing a transaction.
+
+## Decision
+
+### The Phases of a Consumer Chain
```

Comment:
Add a few lines about ownership of consumer chain for Opt In vs Top N. It would add clarity and help explaining who can modify chain params.

Reply:
@insumity any thoughts on this?

Reply:
So added parts referring to the fact that Top N chains are owned by the governance account, while others by the `owner_address` set internally. Feel free to re-open if this did not answer your question.